core/domain/question_services.py

```diff
@@ -285,20 +369,20 @@ def get_skills_linked_to_question(question_id):
     Returns:
         list(Skill). The list of skills that are linked to the question.
     """
-    question_skill_link_models = (
-        question_models.QuestionSkillLinkModel.get_models_by_question_id(
-            question_id))
-    skill_ids = [model.skill_id for model in question_skill_link_models]
-    skills = skill_services.get_multi_skills(skill_ids)
+    question = get_question_by_id(question_id)
+    if not question:
+        raise Exception('Question doesn\'t exist in the backend')
+    skills = skill_services.get_multi_skills(question.linked_skill_ids)
     return skills
 
 
 def update_skill_ids_of_questions(
```

Comment:
Perhaps rename to replace_skill_id_for_all_questions

Reply:
Done